app/src/main/java/com/ozsaat/feeddit/NavigationAdapter.java

```diff
@@ -29,8 +29,8 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(ViewHolder holder, int position) {
-        String item = this.items.get(position);
-        holder.titleTextView.setText(item);
+        NavigationItem item = this.items.get(position);
+        holder.titleTextView.setText((CharSequence) item);
```

Comment:
You can't cast ```NavigationItem``` to a ```CharSequence``` because it does not extend or implement it. You need to get the title from ```NavigationItem``` by calling ```item.getTitle()```